### ISSUE TITLE
[Feature Request] Support get slowlog in redis

### ISSUE THREAD

**Issue by xluffy** [influxdata/telegraf#2565]
## Feature Request

Get slow log command from redis.

Example command:

```
> SLOWLOG GET
 1) 1) (integer) 10595463
    2) (integer) 1490323467
    3) (integer) 8831
    4) 1) "HGETALL"
       2) "telegraf:metric:redis"
```
- A unique progressive identifier for every slow log entry.
- The unix timestamp at which the logged command was processed. (important)
- The amount of time needed for its execution, in microseconds. (important)
- The array composing the arguments of the command. (important)

Ref https://redis.io/commands/slowlog

### Use case:

Monitoring slow query in redis, time duration of command and count slow command

![redis_slow](https://cloud.githubusercontent.com/assets/1505720/24278753/4c64cf36-1078-11e7-95eb-16bb5d41d59a.png)

**Comment by AjithKumar-Raj** [comment]
@danielnelson can I help for this?

**Comment by danielnelson** [comment]
@AjithKumar-Raj Yes, that would be great.  Before getting started on code we should discuss the configuration and the details of the metric model.  Here is my first instinct:
```
[[inputs.redis]]
  # other existing options

  # get the last 10 slowlog entries, if unset or 0 disable slowlog collection
  slowlog = 10
```

Output example, timestamp is the time the command was executed.  Times are all in nanoseconds:
```
redis_slowlog,command=HGETALL,arguments=telegraf:metric:redis execution_ns=1000000000i 1550623426000000000
```

One concern I have is the number of potential series the arguments could create.  If the arguments don't vary to much it shouldn't be too much of a problem, and I think for most people this will be the case.  But it't not hard to think of a usage pattern that could be a problem:
```
redis_slowlog,command=ZRANGE,arguments=telegraf:metric:redis\ 1\ 2 execution_ns=1000000000i 1550623426000000000
redis_slowlog,command=ZRANGE,arguments=telegraf:metric:redis\ 2\ 3 execution_ns=1000000000i 1550623426000000000
```

**Comment by AjithKumar-Raj** [comment]
@danielnelson Our Redis dependency(github.com/go-redis/redis) doesn't support slow log.

**Comment by danielnelson** [comment]
That does complicate things, but it should be possible to implement this and submit it to go-redis.
